package.json

```diff
@@ -82,7 +82,7 @@
 		"schoolImport": "node ./scripts/schoolImport.js"
 	},
 	"dependencies": {
-		"@aws-sdk/lib-storage": "^3.50.0",
+		"@aws-sdk/lib-storage": "^3.100.0",
```

Comment:
Oh ha then we should speak with QA for full check of old file stuff before merging.